apps/admin-portal/src/components/server-configurations/login-policies.tsx

```diff
@@ -0,0 +1,577 @@
+						configs.maxFailedLoginAttemptsToAccountLock = connector.properties.find(
+							property => property.name ==
+								ServerConfigurationsConstants.MAX_FAILED_LOGIN_ATTEMPTS_TO_ACCOUNT_LOCK).value;
+						configs.accountLockTime = connector.properties.find(
+							property => property.name == ServerConfigurationsConstants.ACCOUNT_LOCK_TIME).value;
+						configs.accountLockTimeIncrementFactor = connector.properties.find(
+							property => property.name ==
+								ServerConfigurationsConstants.ACCOUNT_LOCK_TIME_INCREMENT_FACTOR).value;
+						configs.accountLockInternalNotificationManagement = extractArrayValue(connector,
+							ServerConfigurationsConstants.ACCOUNT_LOCK_INTERNAL_NOTIFICATION_MANAGEMENT);
+					} else if (connector.id === ServerConfigurationsConstants.ACCOUNT_DISABLING_CONNECTOR_ID) {
+						configs.accountDisablingEnable = extractArrayValue(connector,
+							ServerConfigurationsConstants.ACCOUNT_DISABLING_ENABLE);
+						configs.accountDisableInternalNotificationManagement = extractArrayValue(connector,
+							ServerConfigurationsConstants.ACCOUNT_DISABLE_INTERNAL_NOTIFICATION_MANAGEMENT);
+					} else if(connector.id === ServerConfigurationsConstants.CAPTCHA_FOR_SSO_LOGIN_CONNECTOR_ID) {
+						const reCaptchaAlwaysEnable = extractArrayValue(connector,
+							ServerConfigurationsConstants.RE_CAPTCHA_ALWAYS_ENABLE);
+						const reCaptchaAfterMaxFailedAttemptsEnable = extractArrayValue(connector,
+							ServerConfigurationsConstants.RE_CAPTCHA_AFTER_MAX_FAILED_ATTEMPTS_ENABLE);
+						if (reCaptchaAlwaysEnable.length > 0) {
+							configs.reCaptchaPreference =
+								ServerConfigurationsConstants.RE_CAPTCHA_ALWAYS_ENABLE;
+						} else if (reCaptchaAfterMaxFailedAttemptsEnable.length > 0) {
+							configs.reCaptchaPreference =
+								ServerConfigurationsConstants.RE_CAPTCHA_AFTER_MAX_FAILED_ATTEMPTS_ENABLE;
+						}
+						configs.maxFailedLoginAttemptsToReCaptcha = connector.properties.find(
+							property => property.name ==
+								ServerConfigurationsConstants.MAX_FAILED_LOGIN_ATTEMPTS_TO_RE_CAPTCHA).value;
+					}
+				});
+				setLoginPoliciesConfigs(configs);
+			});
+	}, []);
+
+	const extractArrayValue = (response, key) => {
+		return response.properties.find(prop => prop.name === key).value === "true" ? [key] : [];
+	};
+
+	const showLoginPoliciesSummary = (
+		<Forms>
+			<Grid padded={ true }>
+				<Grid.Row columns={ 1 }>
+					<Grid.Column mobile={ 16 } tablet={ 16 } computer={ 14 }>
+						<Field
+							name={ ServerConfigurationsConstants.ACCOUNT_LOCK_ENABLE }
+							required={ false }
+							requiredErrorMessage=""
+							type="checkbox"
+							children={ [
+								{
+									label: t("devPortal:components.serverConfigs.loginPolicies." +
+										"accountLock.form.accountLockEnable.label"),
+									value: ServerConfigurationsConstants.ACCOUNT_LOCK_ENABLE
+								}
+							] }
+							value={ loginPoliciesConfigs.accountLockEnable }
+							listen={
+								(values) => {
```

Comment:
Check for `max-length` issues. There might be one here.

Reply:
All checked.